ui/page/root/overview_page.go

```diff
@@ -40,20 +44,27 @@ type OverviewPage struct {
 	infoButton, forwardButton cryptomaterial.IconButton // TOD0: use *cryptomaterial.Clickable
 	coinSlider                *cryptomaterial.Slider
 	mixerSlider               *cryptomaterial.Slider
-
-	proposalItems []*components.ProposalItem
-	orders        []*instantswap.Order
+	proposalItems             []*components.ProposalItem
+	orders                    []*instantswap.Order
+	sliderRedirectBtn         *cryptomaterial.Clickable
 
 	card cryptomaterial.Card
+
+	dcr *assetSliderItem
+	btc *assetSliderItem
+	ltc *assetSliderItem
+
+	assetsTotalBalance map[libutils.AssetType]sharedW.AssetAmount
+	usdExchangeRate    float64
 }
 
-type supportedCoinSliderItem struct {
-	Title    string
-	MainText string
-	SubText  string
+type assetSliderItem struct {
```

Comment:
```suggestion
type assetBalanceSliderItem struct {
```

sorry for the back and forth on this